tests/widgets_barchart.rs

```diff
@@ -36,7 +37,67 @@ fn widgets_barchart_not_full_below_max_value() {
         "│       █████████████████████│",
         "│       █████████████████████│",
         "│       ██50█████99█████100██│",
-        "│empty  half   almost full   │",
+        "│ empty  half  almost  full  │",
```

Comment:
@joshka the only thing which is not backwards compatible now is this.

by default the labels are centered. I see this as an improvement.

Reply:
I agree that this is better. It's a small enough breaking change that it doesn't sound like anyone would worry about it.

Would you mind splitting out this one change outside of the barchart group PR?
Doing it that way will mean we can note the breaking change separately in the changelog.
E.g. something like "feat(barchart)!: center labels by default"